src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -157,6 +184,11 @@ class JooqBurdenEstimateRepository(
                 }
             }
         }
+
+        dsl.update(BURDEN_ESTIMATE_SET)
+                .set(BURDEN_ESTIMATE_SET.STATUS, "complete")
+                .where(BURDEN_ESTIMATE_SET.ID.eq(setId))
+                .execute()
```

Comment:
Ah, of course, this is the bit of actually new logic.

Reply:
and yep, here